bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details-utils.ts

```diff
@@ -102,3 +105,23 @@ export function createDataPoints(evaluationHistory: Trace[]): IDataPoint[] {
     .sort((a, b) => a.yElement.localeCompare(b.yElement));
   return [...scores, ...sortedResults];
 }
+
+export function parseSloOfEvaluations(evaluationTraces: Trace[]): void {
+  const unparsedEvaluations = (trace: Trace): trace is Trace & { data: { evaluation: IEvaluationData } } =>
+    !!trace?.data?.evaluation?.sloFileContent && !trace.data.evaluation.sloFileContentParsed;
+
+  for (const evaluationTrace of evaluationTraces.filter(unparsedEvaluations)) {
+    parseSloFile(evaluationTrace.data.evaluation);
+  }
```

Comment:
Why not go with the `forEach` approach, if you're already using the `filter` method?

```suggestion
  evaluationTraces
    .filter(unparsedEvaluations)
    .forEach(evaluationTrace => parseSloFile(evaluationTrace.data.evaluation));
```

Reply:
To be honest, I just don't like the `forEach` syntax. If I need a for-loop without an index, I'll always use a for-loop instead of `forEach`.